.github/templates/common.yml.j2

```diff
@@ -218,11 +204,13 @@ on:
 {%- endmacro -%}
 
 {%- macro checkout(submodules="recursive", deep_clone=True, directory="", repository="pytorch/pytorch", branch="", checkout_pr_head=True) -%}
-      - name Checkout and clean !{{ 'PyTorch' if repository == "pytorch/pytorch" else repository }}
+      - name: Checkout and clean !{{ 'PyTorch' if repository == "pytorch/pytorch" else repository }}
```

Comment:
wow was this code ever even called if it got by being malformed like this

Reply:
I don't know why this is showing up in the diff.  This PR is diffing against this other one which also has the colon 🤷‍♂️
https://github.com/pytorch/pytorch/pull/80462/files#diff-aa3d5c6114c91192e4f7d66f99f244ed95b08d00a140df29110056ae8d0a68a0R221